docs/source/en/kv_cache.md

```diff
@@ -238,6 +238,24 @@ For more examples with Static Cache and JIT compilation, take a look at [StaticC
 "Hello, my name is [Your Name], and I am a [Your Profession] with [Number of Years] of"
 ```
 
+Like [`~OffloadedCache`] exists for offloading a "DynamicCache", there is also an offloaded static cache. Just
```

Comment:
```suggestion
### Offloaded Static Cache

Like [`~OffloadedCache`] exists for offloading a "DynamicCache", there is also an offloaded static cache. Just
```

I think it deserves a subsection of its own 🤗

Reply:
Will do, also, I will add it to the overview table.

Reply:
Done! Although I do wonder what `Initialization Recommended` means in the overview table?

Reply:
Whether it should be init outside `generate` and passed to generate or not cc @zucchini-nlp on this!

Reply:
Some cache classes are recommended to initialize outside generation, e.g. StaticCache with compilation had some issues when we initialized cache while compiling.

Also, some cache types are not handled automatically by our API, e.g. SinkCache so the user has no option as to initialize and pass `past_key_values`